pkg/formatters/plugins.go

```diff
@@ -8,8 +8,8 @@ import (
 )
 
 const (
-	// the pretty output format for plugins requests
-	prettyPlugins = "{{.Name}}\t{{.Network}}\t{{.Address}}\n"
+	// the pretty output for plugins requests
+	prettyPlugins = "{{.Name}}\t{{.Network.Protocol}}\t{{.Network.Address}}\t{{.Version.Version}}\t{{.Health.Status}}\n"
```

Comment:
I think using `{{.Tag}}` instead of `{{.Name}}` here would make sense now, since the tag is just a normalized string that combines the maintainer + name